.azure/pipelines/azure-pipelines-mirror-within-azdo.yml

```diff
@@ -0,0 +1,61 @@
+# Configure which branches trigger builds
+trigger:
+  batch: true
+  branches:
+    include:
+    - internal/release/6.0
+
+variables:
+  - group: Mirror-Credentials
+
+# Merges code from one AzDO branch into another
+jobs:
+- template: /eng/common/templates/jobs/jobs.yml
+  parameters:
+    enableTelemetry: true
+    helixRepo: dotnet/arcade
+    jobs:
+    - job: Merge_Azure_DevOps_Branches
+      enableSBOM: false
+      condition: eq(variables['Build.Reason'], 'IndividualCI')
+      pool:
+        name: NetCore1ESPool-Internal
+        demands: ImageOverride -equals Build.Server.Amd64.VS2019
+      variables:
+      - name: WorkingDirectoryName
+        value: repo-dir
+      - name: AzdoRepo
+        value: dotnet-aspnetcore
+      - name: TargetBranchName
+        value: $(Build.SourceBranch)-nonstable
+      - name: BranchToMirror
+        value: $(Build.SourceBranch)
+      steps:
+      - script: |
+          git clone https://dn-bot:$(dn-bot-dnceng-build-rw-code-rw)@dev.azure.com/dnceng/internal/_git/$(AzdoRepo) $(WorkingDirectoryName) --recursive --no-tags --branch $(TargetBranchName)
+        displayName: Clone AzDO repo
+      - script: |
+          git -c user.email="dotnet-bot@microsoft.com" -c user.name="dotnet-bot" merge origin/$(BranchToMirror) -m "Merge in '$(BranchToMirror)' changes" 
+        displayName: Merge head branch to target branch
+        workingDirectory: $(WorkingDirectoryName)
+      - script: |
+          git push origin $(TargetBranchName)
+        displayName: Push changes to Azure DevOps repo
+        workingDirectory: $(WorkingDirectoryName)
+
+      - task: PowerShell@1
+        displayName: Broadcast target, branch, commit in metadata
+        continueOnError: true
+        condition: always()
+        inputs:
+          scriptType: inlineScript
+          arguments: '$(AzdoRepo) $(BranchToMirror)'
+          workingDirectory: $(WorkingDirectoryName)
+          inlineScript: |
+            param([string]$repo, [string]$branch)
+
+            $commit = (git rev-parse HEAD).Substring(0, 7)
+            $target = "$repo $branch".Replace('/', ' ')
```

Comment:
This script isn't centralized; no need for `$repo` in either the build number or the tag.

Reply:
Removed